### PR TITLE
Function list with lower CCNs that the current ones

### LINKED ISSUE #14: Less complex functions in functions list
The functions in the list with high CCNs have very high CCN which will make this assignment very hard. Since it is permitted in the assignment to use functions with CCN >= 15 we should use such functions that have a lower CCN than the current ones.

### PR DESCRIPTION
**Pull request by alanihre**
Changed the functions in the list of functions with high CCN so that the numbers are lower but higher than 15. Fix #14.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
-     114     55    849      5     119 vars2fortran@3125-3243@numpy/numpy/f2py/crackfortran.py
-     138     55    749      1     139 scaninputline@176-314@numpy/numpy/f2py/f2py2e.py
-     143     55   1295      3     148 getarrlen@2167-2314@numpy/numpy/f2py/crackfortran.py
-     147     58   1177      0     169 run_compile@487-655@numpy/numpy/f2py/f2py2e.py
-     191     58   1158      3     206 readfortrancode@330-535@numpy/numpy/f2py/crackfortran.py
-     140     65   1172      4     142 updatevars@1536-1677@numpy/numpy/f2py/crackfortran.py
-     335    162   3021      1     347 analyzevars@2524-2870@numpy/numpy/f2py/crackfortran.py
-     527    176   3988      3     545 analyzeline@903-1447@numpy/numpy/f2py/crackfortran.py
+     59      18    375      2     150 add_data_files@1184-1333@./distutils/misc_util.py
+     49      17    363      1      57 finalize_options@86-142@./distutils/command/build_src.py
```
Comment (erikjoh) [inline README.md]: https://codecov.io/gh/numpy/numpy/src/master/numpy/distutils/command/build_src.py#L86

File: README.md [hunk 2]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
-     121     50    960      2     126 buildcallback@452-577@numpy/numpy/f2py/cb_rules.py
-     114     55    849      5     119 vars2fortran@3125-3243@numpy/numpy/f2py/crackfortran.py
-     138     55    749      1     139 scaninputline@176-314@numpy/numpy/f2py/f2py2e.py
-     143     55   1295      3     148 getarrlen@2167-2314@numpy/numpy/f2py/crackfortran.py
-     147     58   1177      0     169 run_compile@487-655@numpy/numpy/f2py/f2py2e.py
-     191     58   1158      3     206 readfortrancode@330-535@numpy/numpy/f2py/crackfortran.py
-     140     65   1172      4     142 updatevars@1536-1677@numpy/numpy/f2py/crackfortran.py
-     335    162   3021      1     347 analyzevars@2524-2870@numpy/numpy/f2py/crackfortran.py
-     527    176   3988      3     545 analyzeline@903-1447@numpy/numpy/f2py/crackfortran.py
+     59      18    375      2     150 add_data_files@1184-1333@./distutils/misc_util.py
```
Comment (erikjoh) [inline README.md]: https://codecov.io/gh/numpy/numpy/src/master/numpy/distutils/misc_util.py#L1184

File: README.md [hunk 3]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     49      17    363      1      57 finalize_options@86-142@./distutils/command/build_src.py
+     83      17    493      4      99 _strings_richcompare@1001-1099@./core/src/multiarray/arrayobject.c
```
Comment (erikjoh) [inline README.md]: These are C programs so I don't think we are allowed to use them (all should be python files).

File: README.md [hunk 4]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     123     21    573      3     193 array_richcompare@1352-1544@./core/src/multiarray/arrayobject.c
+     115     18    598      3     132 array_new@1618-1749@./core/src/multiarray/arrayobject.c
+     123     21    573      3     193 array_richcompare@1352-1544@./core/src/multiarray/arrayobject.c
+     98      17    609      3     111 PyArray_ArgMax@41-151@./core/src/multiarray/calculation.c
+     89      18    540      5     119 PyArray_AssignRawScalar@191-309@./core/src/multiarray/array_assign_scalar.c
+     79      17    389     10      99 npyiter_prepare_operands@1160-1258@./core/src/multiarray/nditer_constr.c
+     44      16    382      2      52 _get_formatdict@359-410@./core/arrayprint.py
```
Comment (erikjoh) [inline README.md]: https://codecov.io/gh/numpy/numpy/src/master/numpy/core/arrayprint.py#L385

File: README.md (hunk 3 above)
Comment (erikjoh) [reply README.md]: It seems like we in 2.2.3 have to write our unit tests for these functions.

**Review by erikjoh — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (alanihre) [reply README.md]: Ah I missed that they were C functions so I'll change that.

File: README.md (hunk 1 above)
Comment (alanihre) [reply README.md]: What is the requested change here?

Comment (erikjoh) [reply README.md]: Woops sorry these are not requests. Just reference to the functions codecov so we more easily can write tests for them later that improve coverage :slightly_smiling_face:

Comment (alanihre) [reply README.md]: I see 👍

**Comment by alanihre** [comment]
Request changes are added.

File: README.md [hunk 5]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
-     121     50    960      2     126 buildcallback@452-577@numpy/numpy/f2py/cb_rules.py
-     114     55    849      5     119 vars2fortran@3125-3243@numpy/numpy/f2py/crackfortran.py
-     138     55    749      1     139 scaninputline@176-314@numpy/numpy/f2py/f2py2e.py
-     143     55   1295      3     148 getarrlen@2167-2314@numpy/numpy/f2py/crackfortran.py
-     147     58   1177      0     169 run_compile@487-655@numpy/numpy/f2py/f2py2e.py
-     191     58   1158      3     206 readfortrancode@330-535@numpy/numpy/f2py/crackfortran.py
-     140     65   1172      4     142 updatevars@1536-1677@numpy/numpy/f2py/crackfortran.py
-     335    162   3021      1     347 analyzevars@2524-2870@numpy/numpy/f2py/crackfortran.py
-     527    176   3988      3     545 analyzeline@903-1447@numpy/numpy/f2py/crackfortran.py
+     59     18    375      2     150 add_data_files@1184-1333@./distutils/misc_util.py
```
Comment (erikjoh) [inline README.md:45]: https://codecov.io/gh/numpy/numpy/src/master/numpy/distutils/misc_util.py#L1184

File: README.md [hunk 6]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     61     22    467      2     120 add_data_dir@1054-1173@./distutils/misc_util.py
```
Comment (erikjoh) [inline README.md:46]: https://codecov.io/gh/numpy/numpy/src/master/numpy/distutils/misc_util.py#L1054

File: README.md [hunk 7]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     49     17    363      1      57 finalize_options@86-142@./distutils/command/build_src.py
```
Comment (erikjoh) [inline README.md:47]: https://codecov.io/gh/numpy/numpy/src/master/numpy/distutils/command/build_src.py#L86

File: README.md [hunk 8]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     41     16    311      3      80 CCompiler_customize@468-547@./distutils/ccompiler.py
```
Comment (erikjoh) [inline README.md:48]: https://codecov.io/gh/numpy/numpy/src/master/numpy/distutils/ccompiler.py#L468

File: README.md [hunk 9]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     52     20    420      3      54 get_paths@663-716@./distutils/system_info.py
```
Comment (erikjoh) [inline README.md:49]: https://codecov.io/gh/numpy/numpy/src/master/numpy/distutils/system_info.py#L663

File: README.md [hunk 10]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     65     17    392      1      71 get_atlas_version@1465-1535@./distutils/system_info.py
```
Comment (erikjoh) [inline README.md:50]: https://codecov.io/gh/numpy/numpy/src/master/numpy/distutils/system_info.py#L1465

File: README.md [hunk 11]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     47     17    294      1      63 setup@109-171@./distutils/core.py
```
Comment (erikjoh) [inline README.md:51]: https://codecov.io/gh/numpy/numpy/src/master/numpy/distutils/core.py#L109

File: README.md [hunk 12]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     35     16    272      4      56 _exec_command@265-320@./distutils/exec_command.py
```
Comment (erikjoh) [inline README.md:52]: https://codecov.io/gh/numpy/numpy/src/master/numpy/distutils/exec_command.py#L265

File: README.md [hunk 13]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     38     16    245      2      55 _scalar_str@115-169@./core/_dtype.py
```
Comment (erikjoh) [inline README.md:53]: https://codecov.io/gh/numpy/numpy/src/master/numpy/core/_dtype.py#L115

This one seems to only have one else statement that is not covered. Guess this makes it easy to write a single test for.

File: README.md [hunk 14]
```diff
@@ -36,22 +36,22 @@ Tests can then be run after installation with:
...
+     34     17    238      1      57 long_double_representation@345-401@./core/setup_common.py
```
Comment (erikjoh) [inline README.md:54]: https://codecov.io/gh/numpy/numpy/src/master/numpy/core/setup_common.py#L345

**Review by erikjoh — APPROVED** [review]

**Pull request merged**